src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService users = new UserService();
+        boolean result = false;
+        if (users.findByEmail(email) == null) {
+            return false;
+        }
+        if (password.equals(users.findByEmail(email).getPassword())) {
```

Comment:
in this impl you search for the user twice. Instead, search once and save the result to a User user local variable